x-pack/plugins/data_usage/public/app/components/data_usage_metrics.tsx

```diff
@@ -6,29 +6,24 @@
  */
 
 import React, { useCallback, useEffect, useState } from 'react';
-import {
-  EuiTitle,
-  EuiSpacer,
-  EuiFlexGroup,
-  EuiFlexItem,
-  EuiLoadingElastic,
-  EuiPageSection,
-  EuiText,
-} from '@elastic/eui';
-import { FormattedMessage } from '@kbn/i18n-react';
-import { i18n } from '@kbn/i18n';
-import { UsageMetricsRequestSchemaQueryParams } from '../../common/rest_types';
-import { Charts } from './components/charts';
-import { UsageMetricsDateRangePicker } from './components/date_picker';
-import { useBreadcrumbs } from '../utils/use_breadcrumbs';
-import { useKibanaContextForPlugin } from '../utils/use_kibana';
-import { PLUGIN_NAME } from '../../common';
-import { useGetDataUsageMetrics } from '../hooks/use_get_usage_metrics';
-import { DEFAULT_DATE_RANGE_OPTIONS, useDateRangePicker } from './hooks/use_date_picker';
-import { useDataUsageMetricsUrlParams } from './hooks/use_charts_url_params';
-import { MetricsResponse } from './types';
+import { css } from '@emotion/react';
+import { EuiFlexGroup, EuiFlexItem, EuiLoadingElastic } from '@elastic/eui';
+import { Charts } from './charts';
+import { useBreadcrumbs } from '../../utils/use_breadcrumbs';
+import { useKibanaContextForPlugin } from '../../utils/use_kibana';
+import { PLUGIN_NAME } from '../../../common';
+import { useGetDataUsageMetrics } from '../../hooks/use_get_usage_metrics';
+import { useDataUsageMetricsUrlParams } from '../hooks/use_charts_url_params';
+import { MetricsResponse } from '../types';
+import { DEFAULT_DATE_RANGE_OPTIONS, useDateRangePicker } from '../hooks/use_date_picker';
+import { UsageMetricsRequestSchemaQueryParams } from '../../../common/rest_types';
+import { ChartFilters } from './filters/charts_filters';
 
-export const DataUsage = () => {
+const EuiItemCss = css`
+  width: 100%;
+`;
+
+export const DataUsageMetrics = () => {
```

Comment:
`useGetDataUsageDataStreams` needs to be called somewhere on this page first because we have to get the top 10 by default and have them selected by default when the user first lands. `dataStreams` will be required for the `/metrics` endpoint so getting the data streams has to happen first.  It can be passed down as props to the `ChartFilters`?  Does that make sense?  Ideally we can call them in parallel but I think we agreed this simplifies things and avoids us having to "get all" data streams in `/metrics`.

Reply:
Actually it won't be called here. I've a hook to populate the data streams filter (`use_charts_filter.tsx`) which should do the selection logic based on storage sizes. I'll addd that change